packages/assistant-stream/src/core/serialization/data-stream/DataStream.ts

```
@@ -0,0 +1,347 @@
+              break;
+
+            case DataStreamStreamChunkType.Error:
+              controller.enqueue({
+                type: "error",
+                path: [],
+                error: value,
+              });
+              break;
+
+            default:
+              const exhaustiveCheck: never = type;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)